safenode/src/node/api.rs

```diff
@@ -327,9 +324,10 @@ impl Node {
 
     async fn send_to_closest(&self, request: &Request) -> Result<Vec<Result<Response>>> {
         info!("Sending {:?} to the closest peers.", request.dst());
+        // todo: if `self` is present among the closest peers, the request should be routed to self?
```

Comment:
Could be.

It can be handled like that, or that we handle it directly. We have used both patterns. I'm on the fence on what is cleanest.